frontends/web/src/locales/en/app.json

```diff
@@ -216,9 +216,9 @@
     "noPasswordMatch": "Passwords did not match, please try again.",
     "pairing": {
       "failed": "Unconfirmed pairing. Please replug your BitBox02.",
-      "paired": "You have confirmed on your device that the code matches. If this is correct, you can continue by clicking the button below.",
+      "paired": "Please ensure you verified the pairing code on the device and continue.",
```

Comment:
imo I change to "You have confirmed the following code on your device. Please continue."

Reason is because I think the instructions are not longer clear since you already confirmed on the device. The important part is that the code is still displayed.  Because if a  MITM simulating the BB02 simulated a "confirm" on the device, the BB02 will still show the pairing code (since it was not actually confirmed yet. Then it is clear the user something is wrong because the codes don't match.

Reply:
updated pr and locize